File: Programming-languages-and-laboratory/Java/Exercises/26_compito_java_2019_11_19/Partecipante.java
import java.util.*;

public abstract class Partecipante {

	protected int codice;
	protected String nome, cognome, tipo;

	public Partecipante(int codice, String nome, String cognome, String tipo) {
		this.codice = codice;
		this.nome = nome;
		this.cognome = cognome;
		this.tipo = tipo;
	}

	public String getTipo() {
		return tipo;
	}

	public int getCod() {
		return codice;
	}

	public String getCognome() {
		return cognome;
	}

	public String toString() {
		return tipo + "\t" + codice + "\t" + nome + "\t" + cognome + "\t";
	}
}

File: Programming-languages-and-laboratory/Python/Exercises/26_compito_java_20181119/Partecipante.py
class Partecipante:
    def __init__(self, codice, tipo, nome, cognome):
        self._codice = codice
        self._tipo = tipo
        self._nome = nome
        self._cognome = cognome

    def get_codice(self):
        return self._codice

    def get_tipo(self):
        return self._tipo

    def get_cognome(self):
        return self._cognome

    def __str__(self):
        return self._tipo + "\t" + str(self._codice) + "\t" + self._nome + "\t" + self._cognome
